backend/src/db/daos/noteDao.js

```diff
@@ -1,14 +1,36 @@
 import Note from "../models/note";
 import Group from "../models/group";
 
+/**
+ * Checks if a user is in a group
+ * @param {String} groupId group ID
+ * @param {String} email email of the user
+ * @returns role of the user in the group
+ * @returns null if user is not in group
+ */
+const checkRole = async (groupId, email) => {
```

Comment:
nit: more appropriate name would be something along the lines of "getRole" or "findRole".